package.json

```diff
@@ -3,7 +3,14 @@
   "version": "1.4.0",
   "description": "React Sample Applications for Okta",
   "scripts": {
-    "test": "eslint -c .eslintrc.json --ext .js --ext .jsx okta-hosted-login/src/ custom-login/src/"
+    "test": "eslint -c .eslintrc.json --ext .js --ext .jsx okta-hosted-login/src/ custom-login/src/ && npm run test:e2e",
+    "okta-hosted-login-server": "npm start --prefix okta-hosted-login/",
+    "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
+    "custom-login-server": "npm start --prefix custom-login/",
+    "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
+    "test:e2e": "npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
+    "get-oidc-tck": "rm -rf okta-oidc-tck && git clone https://github.com/okta/okta-oidc-tck.git && git --git-dir=okta-oidc-tck/.git --work-tree=okta-oidc-tck checkout e2eTests",
```

Comment:
This section needs to be removed after `e2eTests` branch is merged into `okta-oidc-tck` repo

Reply:
@robertjd - I've addressed your concern from the old PR here (This will fail with `fatal: destination path 'okta-oidc-tck' already exists and is not an empty directory` if the repo has already been cloned)
I'm deleting the existing `okta-oidc-tck` directory before cloning. Hope that works.

Reply:
I think a better solution is to only clone the repo if it hasn't already been cloned.  This is working well for me:

`"get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git; git --git-dir=okta-oidc-tck/.git --work-tree=okta-oidc-tck checkout e2eTests",`

The general feedback here is: let's make sure we make it easy to run the tests locally as well, so that developers can sanity check locally before needing to wait on travis.

Reply:
Updated the PR to do this.